src/storage-preview/azext_storage_preview/vendored_sdks/azure_mgmt_storage/v2018_07_01/models/custom_domain_py3.py

```diff
@@ -32,7 +32,7 @@ class CustomDomain(Model):
 
     _attribute_map = {
         'name': {'key': 'name', 'type': 'str'},
-        'use_sub_domain': {'key': 'useSubDomain', 'type': 'bool'},
+        'use_sub_domain': {'key': 'useSubDomainName', 'type': 'bool'},
     }
 
     def __init__(self, *, name: str, use_sub_domain: bool=None, **kwargs) -> None:
```

Comment:
Can you also change this model's field to be `use_sub_domain_name` like: https://github.com/Azure/azure-sdk-for-python/pull/3851/files#diff-405ddd0c1a1405b64a5627a8ad998583R41?
This way, when the new sdk is released and vendored through the extension, the `update_storage_account()` code will not need to be updated again.